gensim/models/base_any2vec.py

```diff
@@ -580,6 +580,8 @@ def __init__(self, sentences=None, input_streams=None, workers=3, vector_size=10
             The beginning learning rate. This will linearly reduce with iterations until it reaches `min_alpha`.
         window : int, optional
             The maximum distance between the current and predicted word within a sentence.
+        symmetric : {0, 1}, optional
```

Comment:
Why only symmetric + only left, can we extend this (since we still add a new attribute for it)?

Reply:
CC @gojomo

Reply:
We had a PR with a more general solution, that allowed you to input arbitrary word pairs (so it's up to you how you construct these "contexts"). The left/right context was just one special case.

I think the PR wasn't merged and you later closed it, and I cannot find it now. This was maybe 3 years ago.

Reply:
Um, if it was 3 years ago and I close it  - this PR was abandoned a long time ago (several years), it is already completely irrelevant (but I would like to know what approach was used there), because I prefer more general solution.